source/nodejs/adaptivecards-site/schema-explorer-toc.yml

```diff
@@ -21,6 +21,7 @@
   - Action.ShowCard
   - Action.ToggleVisibility
   - TargetElement
+  - Action.Execute
```

Comment:
We should also include the new `Refresh` and `Authentication` types at the bottom of this file, under `Types`, so users can view the schema for those

Reply:
That's a separate thing though not? This PR only introduces the 1.4 schema.